qwik-graphql-tailwind/src/routes/layout.tsx

```diff
@@ -20,9 +20,12 @@ export default component$(() => {
     }
   });
 
-  if (!store.access_token) {
-    return <div>Loading...</div>;
-  }
```

Comment:
Is the layout a child of any parent? if not, you can't access the store value from context

Reply:
My other pr fixed this, as it is a bug from Qwik.